pkg/kubelet/volumes.go

```diff
@@ -134,6 +142,25 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap,
 		if err != nil {
 			return nil, err
 		}
+		if hasFSGroup && builder.SupportsOwnershipManagement() {
+			filepath.Walk(builder.GetPath(), func(path string, info os.FileInfo, err error) error {
+				if err != nil {
+					return err
+				}
+
+				err = kl.chownRunner.Chown(path, 0, int(fsGroup))
```

Comment:
It's unfortunate that Go does not have a way (as far as I can tell) to change group without also changing ownership - am I missing something?